src/utils.ts

```diff
@@ -155,16 +155,27 @@ export const getBaseAssetIconUrl = (chainId: string) => {
   return 'https://static.cdnlogo.com/logos/e/81/ethereum-eth.svg';
 };
 
-export function graphQlCall(url: string, query: string) {
+export function graphQlCall(
+  url: string,
+  query: string,
+  options: any = {
+    headers: {}
+  }
+) {
   return axios({
     url: url,
     method: 'post',
     headers: {
-      'Content-Type': 'application/json'
+      'Content-Type': 'application/json',
+      ...options.headers
     },
     timeout: 5e3,
     data: {
       query
     }
   });
 }
+
+export function removeFalsyValues(obj: Record<string, any>) {
+  return Object.fromEntries(Object.entries(obj).filter(Boolean));
```

Comment:
> Isn't Boolean false a valid value?
No looks like we can only pass non-empty strings in headers

> It's "just" a type issue
Better to avoid the use of `@ts-ignore` or `as` if we want to take typescript seriously 😄 , we can use them if there is no other way to do something

But I think we could go with this additional filter.

Reply:
you see any other way?

Reply:
Let's merge #270 first, then we can always pass just `{ headers: { 'x-api-key': process.env.HUB_API_KEY } }` as this function second argument without any filter/logic.

Then we can handle the filtering of `undefined` values only in this function.

Reply:
Axios doesn't clean undefined, it fails with 

```bash
TypeError [ERR_HTTP_INVALID_HEADER_VALUE]: Invalid value "undefined" for header "x-api-key"
    at ClientRequest.setHeader (node:_http_outgoing:647:3)
    at new ClientRequest (node:_http_client:284:14)
    at originalMethod (node:http:97:10)
```